DataManipulation/Sorter.cs

```diff
@@ -0,0 +1,46 @@
+﻿using ItemLibrary;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace DataManipulation
+{
+    class Sorter
+    {
+        private List<Item> _items { get; set; }
```

Comment:
atsakant į užduotus pasvarstymus manyčiau, kad turėtum pakeisti fieldą į public. tokiu atveju pasikeitus prekėms (tarkim prekė A removinama iš listo, o prekė B pridedama) nereikės kurtis naujo objekto, o užteks paupdainti esamą priskyrus šios klasės Items laukui naujai sugeneruotą listą. Taip manau išvengsim atvejų kai susikuriam Sorter'į su juo kažką veikiam, tada pasikeičia prekių listas ir senasis Sorter'is mums jau nebebus naudingas, nes bus outdated ir reikės kurtis naują

Reply:
arba dar kilo pasiūlymas pridėt public metodą updateList(list<Item>) tarkim, kuris leistų updatinti listą, bet neleistų keisti to fieldo reikšmės tiesiogiai kreipiantis Sorter.Items

Reply:
Padarysiu public metodą